src/libraries/Microsoft.PowerFx.Core/Glue/SimpleResolver.cs

```diff
@@ -42,20 +42,24 @@ internal class SimpleResolver : INameResolver
 
         IExternalEntity INameResolver.CurrentEntity => null;
 
+        public bool SuggestUnqualifiedEnums { get; private set; }
```

Comment:
>public bool SuggestUnqualifiedEnums { get; private set; } [](http://example.com/codeflow?start=8&length=57)

This can just be fixed: 

public bool SuggestUnqualifiedEnums => true;

Reply:
Canvas uses a different resolver instance.

Reply:
Right!